README.md

```diff
@@ -1 +1,147 @@
 # Awesome-Books-with-Modules
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 [Awesome-Books-with-Modules] <a name="about-project"></a>
+**[Awesome-Books-with-Modules]** is the first project at Microverse that is meant to test one's knowledge on gitflow and Es Linter Configuration. This project was build using Html and css. Git, Github, and Vs Code Studio were key technological tools used in this project.
+
+## 🛠 Built With <a name="built-with"> </a>
+This project is built using pure Html and css
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [a link to the online version]()
+
+- [a link to a presentation about this project]()
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+You need to have a code editor pre-installed (preferably Vs Studio Code Editor). You may also need to install Git Bash terminal but if you're using Vs Code Studio you can use its terminal.
+
+
+### Setup
+
+Step 1:if you don't have git installed on your machine you can download it from [here](https://git-scm.com/downloads).
+
+Once you have git installed on your machine you can clone your project by running the command below to clone your project to your local machine
+
+`git clone <your project link>`
```

Comment:
- [ ] [OPTIONAL] Again to make it easier for people to use this project, I think it will be better if you provide the git clone link for your project which is `https://github.com/Kidd254/Awesome-Books-with-Modules.git`